src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -133,6 +133,9 @@ public static bool IsMsBuildBasedProject(string projectFullPath)
                 argumentBuilder.Append(" /p:RestoreGraphOutputPath=");
                 AppendQuoted(argumentBuilder, resultsPath);
 
+                // Disallow the import of targets/props from packages
+                argumentBuilder.Append(" /p:ExcludeRestorePackageImports=true ");
```

Comment:
>ExcludeRestorePackageImports [](start = 44, length = 28)

Will this be on by default for dotnet restore3? If not, why not? This is a safer default right?